### PR TITLE
[DOCFIX] Improve Basic-Logging doc

### PR DESCRIPTION
**Pull request by jiacheliu3**
1. Some classes like `InodeSyncStream` uses TRACE level log, mention that.
2. Add a way to add a separate logger for certain classes. This can be helpful in many scenarios.

### CODE REVIEW

File: docs/en/operation/Basic-Logging.md [hunk 1]
```diff
@@ -262,3 +269,26 @@ FREE_BYTES, CAPACITY_BYTES]) returned BlockMasterInfo{capacityBytes=11453246122,
...
+1. Clearly separate the wanted logs for further analysis.
+1. Avoid `master.log` or `worker.log` being too big or have too many files created by log rotation.
+1. Use a separate logger to send logs to a remote endpoint like a socket.
+
+This can be achieved by adding a separate logger in the `conf/log4j.properties`.
+For example, the below example redirects debug logs of `alluxio.master.StateLockManager` to a separate set of files,
+so the `master.log` will not be full of the DEBUG logs created by `alluxio.master.StateLockManager`.
+
+```properties
+log4j.category.alluxio.master.StateLockManager=DEBUG, slm
```
Comment (LuQQiu) [inline docs/en/operation/Basic-Logging.md:286]: what's the meaning of slm?

Comment (jiacheliu3) [reply docs/en/operation/Basic-Logging.md:286]: `slm` just means `StateLockManager`, do you have a better name in mind? @LuQQiu

Comment (jiacheliu3) [reply docs/en/operation/Basic-Logging.md]: updated

**Review by LuQQiu — APPROVED** [review]

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**